lib/puppet/provider/selinux_fcontext/semanage.rb

```diff
@@ -13,9 +15,7 @@
   osversion       = Facter.value('operatingsystemmajrelease')
   operatingsystem = Facter.value('operatingsystem')
   @old_semanage = false
-  if (osfamily == 'RedHat') && (Puppet::Util::Package.versioncmp(osversion, '6') <= 0) && (operatingsystem != 'Amazon')
-    @old_semanage = true
-  end
+  @old_semanage = true if (osfamily == 'RedHat') && (Puppet::Util::Package.versioncmp(osversion, '6') <= 0) && (operatingsystem != 'Amazon')
```

Comment:
If you're dropping EL6 support, does that mean this is now redundant?

Reply:
you're correct. I cherry-picked and fixed it in https://github.com/voxpupuli/puppet-selinux/pull/368